logging/logging.go

```diff
@@ -933,11 +926,15 @@ func (l *Logger) toLogEntry(e Entry) (*logpb.LogEntry, error) {
 			e.TraceSampled = e.TraceSampled || traceSampled
 		}
 	}
+	req, err := fromHTTPRequest(e.HTTPRequest)
+	if err != nil {
+		l.client.error(err)
```

Comment:
What is the full flow of how this would be triggered? I want to make sure I understand. Does this error come up intermittently depending on the request? Or would it be a fundamental issue in the user's code (where if it's broken at all, it will never work)?

Reply:
This most often happens when a user makes a mistake, e.g.: 

```go
func indexHandler(w http.ResponseWriter, r *http.Request) {
   ...
    httpLog := logging.Entry{
	HTTPRequest : &logging.HTTPRequest {
		Request : // does not pass in r
	},
   }
   ... 
}
```

cmiiw, I don't think it'll come up intermittently depending on request, even if the request body is nil (the request itself isn't)